index.html

```diff
@@ -75,6 +75,14 @@ <h2 class="section__title">Kontakty</h2>
         <a class="refs__links__link" href="https://twitter.com/klubFITpp"><i class="refs__links__link__logo ion ion-logo-twitter"></i> @klubFITpp</a>
         <a class="refs__links__link" href="mailto:fitpp@su.cvut.cz"><i class="refs__links__link__logo ion ion-ios-mail"></i> fitpp@su.cvut.cz</a>
       </article>
+      <article class="club__board">
```

Comment:
Dle BEM by toto byl element `board` uvnitr prvku `club`. Prejmenoval bych to treba jen na `board` nebo tak neco..